scripts/curvedeformereditor/mainview.py

```diff
@@ -13,8 +13,7 @@
 
 
 def icon(filename):
-    path = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)), 'icons', filename)
+    path = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir, 'icons', filename))
```

Comment:
can you respect the 79 char pep8 rule ? :)

Reply:
for sure man, not sure why my linter didn't bock about this.